packages/bundles-dynamic/src/components/bundle-prices/bundle-prices.js

```diff
@@ -16,16 +16,15 @@ import messages from './messages';
 
 const BundlePrices = ({ match, id, categories, dynamicPrice }) => {
   const intl = useIntl();
-  const { environment, project } = useApplicationContext();
-  const { mcURL } = environment;
+  const { project } = useApplicationContext();
   const { currencies } = project;
   const [currency, setCurrency] = useState(currencies[0]);
   const [country, setCountry] = useState(null);
   const [customerGroup, setCustomerGroup] = useState(null);
   const [channel, setChannel] = useState(null);
   const [date, setDate] = useState('');
 
-  const MC_URL = `https://${mcURL}/${match.params.projectKey}`;
+  const MC_URL = `${window.location.origin}/${match.params.projectKey}`;
```

Comment:
Do you plan to do the refactoring of the routing as a follow up PR? I'm referring to removing the usage of `window.open`.

Reply:
Yes, that's the plan. Created sprint ticket in backlog: https://jira.commercetools.com/browse/PSS-515